.eslintrc.js

```diff
@@ -5,12 +5,22 @@ module.exports = {
     "prettier"
   ],
   rules: {
-    "no-var": 0,
-    "max-len": 0,
+    "max-len": "off",
+    "strict": "error",
```

Comment:
I noticed some files were using directives and some weren't, and I'm assuming we want to be in strict mode for all files.